docs/outposts/OutpostL2.md

```diff
@@ -0,0 +1,291 @@
+# Outpost L2 contracts
```

Comment:
This content does not make much sense since it is repeated in the SC section.
This should be a runbook on what is an outpoust ? how to create one ? paraneters involved ? ...